geomstats/_backend/numpy/linalg.py

```diff
@@ -2,17 +2,18 @@
 
 import numpy as np
 import scipy.linalg
-from numpy.linalg import (  # NOQA
+from numpy.linalg import (
     cholesky,
     det,
     eig,
     eigh,
     eigvalsh,
     inv,
     norm,
-    solve,
     svd,
-)
+    matrix_rank,
```

Comment:
This should be sorted in alphabetic order (to avoid importing the same function twice)